home/models.py

```diff
@@ -74,6 +79,67 @@ def is_future(self):
     def accepting_rsvps(self):
         return self.is_future and self.status == self.SCHEDULED
 
+    def add_participant_email_verification(self, user):
+        self.rsvped_members.add(user.id)
+        if not user.email:
+            return
+
+        email_dict = {
+            "event" : self,
+            "user": user,
+        }
+
+        subject = 'Djangonaut Space RSVP'
+        text_message = render_to_string('email/email_rsvp.txt', email_dict)
+        html_message = render_to_string('email/email_rsvp.html', email_dict)
+
+        email_kwargs = {
+            'subject': subject,
+            'body': text_message,
+            'from_email': settings.DEFAULT_FROM_EMAIL,
+            'to': [user.email],
+            'reply_to': [settings.DEFAULT_FROM_EMAIL,],
+        }
+
+        email = EmailMultiAlternatives(**email_kwargs)
+        email.attach_alternative(html_message, "text/html")
+        email.send()
```

Comment:
As an FYI,[ `send_mail` supports `html_message`](https://docs.djangoproject.com/en/4.2/topics/email/#send-mail) avoiding the need for these three lines.